themes/digital.gov/layouts/partials/core/home/news_featured.html

```diff
@@ -11,105 +11,94 @@ <h2>News and Events</h2>
             <p>Innovative work, news, and ideas from people and teams in government</p>
 
             <div class="join-buttons">
-              <a href="{{- "/join" | absURL -}}">Write for us</a>
+              <a href="{{- "/join" | relURL -}}">Write for us</a>
```

Comment:
This was being flagged as a warning in dev tools. It recommended using relative paths for internal links.